src/prometheus/index.ts

```diff
@@ -0,0 +1,22 @@
+import { createGauge, type Gauge } from './createGauge';
+import { createCounter } from './createCounter';
+
+export * from './createCounter';
+export * from './createGauge';
+export * from './createSummary';
+export * from './createHistogram';
+
+export const lastMetricsUpdate: Gauge = createGauge({
+    name: 'last_metrics_update_epoch_timestamp_ms',
+    help: 'An epoch timestamp (in milliseconds) set to when our unleash-client last got an update from upstream Unleash',
+});
+
+export const lastMetricsFetch: Gauge = createGauge({
+    name: 'last_metrics_fetch_epoch_timestamp_ms',
+    help: 'An epoch timestamp (in milliseconds) set to when our unleash-client last checked (regardless if there was an update or not)',
+});
+
+createCounter({
+    name: 'unleash_proxy_up',
+    help: 'Indication that the service is up.',
+}).inc(1);
```

Comment:
The gist of the PR is here. Copies Nuno's great work on creating typechecked labels (though we don't actually use labels here) from unleash/unleash, but sets up for the future. Could consider removing histogram and summary, since it's currently unused.